package.json

```diff
@@ -15,7 +15,8 @@
     "start": "tsdx watch",
     "build": "tsdx build",
     "test": "tsdx test --env=jsdom",
-    "lint": "tsdx lint"
+    "lint": "tsdx lint",
+    "prepare": "npm run build"
```

Comment:
Could you explain why this is needed?

Reply:
So when we fork a git repository and want to use that fork in our apps, `npm/yarn` would crib because when the app runs the `npm install` (or yarn install), it won't find the `dist` folder as the fork repository only has source files checked in (`dist` is gitignored). For the install step to also create the build files in `dist/`, we need to indicate it to the package manager using this `prepare` script.